common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -129,7 +129,8 @@ function(HtmlUtils) {
          * initial configuration.
          */
         render: function() {
-            var container = this.el.find('.volume-slider');
+            var container = this.el.find('.volume-slider'),
+                instructionsId = 'volume-instructions-' + this.state.id;
 
```

Comment:
@arizzitano For my own background knowledge: won't this read out something funky on screen readers? Is that ok to do?